bridges/PresidenciaPTBridge.php

```diff
@@ -19,27 +43,36 @@ class PresidenciaPTBridge extends BridgeAbstract {
 		'novembro',
 		'dezembro');
 
+	public function getIcon(){
+		return 'https://www.presidencia.pt/Theme/favicon/apple-touch-icon.png';
+	}
+
 	public function collectData() {
-		$html = getSimpleHTMLDOM($this->getURI() . '/atualidade/mensagens')
-			or returnServerError('Could not load content');
+		foreach(array_keys($this->getParameters()['Section']) as $k) {
+			Debug::log('Key: ' . var_export($k, true));
+			if($this->getInput($k)) {
+				$html = getSimpleHTMLDOMCached($this->getURI() . $k)
+					or returnServerError('Could not load content');
```

Comment:
Recently realised that combination like `getContents() or returnServerError()` does not execute second statement if error happens. Only if returned body is empty. Is that statement for empty body check?

Reply:
Nope, it's to detect any error, just like in most other bridges.

Does that mean that is redundant, that errors on the request just fail the entire thing? Or do we need some extra code for that? Should that be done on the common bridge class?

Reply:
> Does that mean that is redundant, that errors on the request just fail the entire thing?

Yes. If server returns non-200 result code, exception will be raised. For me this is correct behavior, including your case.

If you need to handle errored responses, you can wrap to try catch, but in this case  "Could not load content" seem to be less informative that "Error %HTTP_CODE%, Unexpected response from upstream". Like here: https://feed.eugenemolotov.ru/?action=display&bridge=Vk&u=idasfsafasf1&format=Html

Reply:
Thanks for this, I'll change this PR later. I'll consider this in the future.